ext/RMagick/rmimage.c

```diff
@@ -7255,12 +7255,22 @@ has_attribute(VALUE self, MagickBooleanType (attr_test)(const Image *, Exception
  *
  * @param self this object
  * @return true if image is gray, false otherwise
- * @see has_attribute
  */
 VALUE
 Image_gray_q(VALUE self)
 {
-    return has_attribute(self, (MagickBooleanType (*)(const Image *, ExceptionInfo *))IsGrayImage);
```

Comment:
When I test this with `IMAGEMAGICK_VERSION=6.8.9-10` the test passes. Looking over the `ImageMagick` source, it looks like maybe `IsGrayImage` was replaced with `IsImageGray`? Should we try that instead?